### PR TITLE
feat(origin-ui-core): device import page

### PR DESCRIPTION
**Pull request by marcin-l-tsh**
New Page's UI:
Screenshot 2021-01-22 at 09 05 19
Screenshot 2021-01-22 at 09 06 24

### CODE REVIEW

File: packages/localization/src/languages/en.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
                 "users": "Manage Users"
             },
             "devices": {
+                "import": "Device import",
```
Comment (alexworker23) [inline packages/localization/src/languages/en.json:17]: Changes to localization files, should be as a separate commit to this package.

File: packages/origin-ui-core/src/components/devices/import/ImportDeviceModal.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+const useStyles = makeStyles({
+    modalContent: {
+        display: 'flex',
+        width: '100%',
+        height: '100%',
+        alignItems: 'center',
+        pointerEvents: 'none'
+    },
+    modalWindow: {
+        background: '#434343',
```
Comment (alexworker23) [inline packages/origin-ui-core/src/components/devices/import/ImportDeviceModal.tsx]: Please, avoid using hard coded color values. 
For getting colors use `useOriginConfiguration()` - and get colors from there. 
To modify the color use the `LightenColor` function.
If you follow this strategy - your colors will be auto-adjusted on color-scheme change.

File: packages/origin-ui-core/src/components/devices/import/ImportDevices.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+import { makeStyles } from '@material-ui/core/styles';
+import { CustomFilterInputType, FiltersHeader, ICustomFilterDefinition } from '../../Table';
+import { IDevice, ImportDeviceItem } from './importDeviceItem';
+import { useTranslation } from '../../../utils';
+import { ImportDeviceModal } from './ImportDeviceModal';
+import { Pagination } from '@material-ui/lab';
+
+const useStyles = makeStyles({
+    box: {
+        background: '#2d2d2d',
```
Comment (alexworker23) [inline packages/origin-ui-core/src/components/devices/import/ImportDevices.tsx]: Same here, please avoid hard-coded colors.

File: packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    id: number;
+    name: string;
+    country: string;
+    capacity: number;
+    imported?: boolean;
+}
+
+const useStyles = makeStyles({
+    device: {
+        background: '#373737',
```
Comment (alexworker23) [inline packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx]: hard-coded color

File: packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    button: {
+        borderColor: '#894EC5'
+    },
+
+    buttonLabel: {
+        color: '#894EC5',
+        textTransform: 'uppercase'
+    }
+});
+
```
Comment (alexworker23) [inline packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx]: Hard-coded colors on styles above. 
Also, please, start the name of the file containing component with a capital letter.

File: packages/origin-ui-core/src/components/devices/deviceMenuCreator.tsx [hunk 6]
```diff
@@ -56,6 +57,13 @@ export const deviceMenuCreator = (user: IUser, t: TFunction): IDeviceMenuItem[]
         user?.organization && isRole(user, Role.OrganizationDeviceManager, Role.OrganizationAdmin);
 
     return [
+        {
+            key: 'import',
+            label: t('navigation.devices.import'),
+            component: ImportDevice,
+            features: [OriginFeature.Devices],
```
Comment (alexworker23) [inline packages/origin-ui-core/src/components/devices/deviceMenuCreator.tsx]: This menu item, for now should be hidden under specific feature flag, like `OriginFeature.DeviceImport`. 
And this feature should be disabled in `OriginConfigurationContext` for now.

File: packages/localization/src/languages/en.json (hunk 1 above)
Comment (marcin-l-tsh) [reply packages/localization/src/languages/en.json:17]: done

File: packages/origin-ui-core/src/components/devices/import/ImportDeviceModal.tsx (hunk 2 above)
Comment (marcin-l-tsh) [reply packages/origin-ui-core/src/components/devices/import/ImportDeviceModal.tsx]: replaced with config variable

File: packages/origin-ui-core/src/components/devices/import/ImportDevices.tsx (hunk 3 above)
Comment (marcin-l-tsh) [reply packages/origin-ui-core/src/components/devices/import/ImportDevices.tsx]: fixed

File: packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx (hunk 4 above)
Comment (marcin-l-tsh) [reply packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx]: fixed

File: packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx (hunk 5 above)
Comment (marcin-l-tsh) [reply packages/origin-ui-core/src/components/devices/import/importDeviceItem.tsx]: renamed

File: packages/origin-ui-core/src/components/devices/deviceMenuCreator.tsx (hunk 6 above)
Comment (marcin-l-tsh) [reply packages/origin-ui-core/src/components/devices/deviceMenuCreator.tsx]: done

**Review by alexworker23 — APPROVED** [review]
